### DISCUSSION TITLE
If I want to use trilium professionally and correctly, should I have a server to use?

### DISCUSSION THREAD

**Discussion by asasn** [TriliumNext/Trilium#4843] [General]
I read:
`Trilium stores notes in a document which is an SQLite database. `

The reason for not supporting synchronization features (such as webdav)?

Please forgive me that I am just a user who is used to Windows system and exe file, using trilium-windows-x64-0.63.7.zip, if I want to use it on both Android and Windows, Should I abandon trilium-windows-x64-0.63.7.zip in favor of docker deployment or trilium-linux-x64-server-0.63.7.tar.xz?And then use a browser to visit the website and open it like that?

Trying to understand this software...

**Comment by grotesque** [comment]
i've the same question. How should we proceed?
Abandon trilium-windows-xxx.zip in favor of linux pacakge server.
Then access that server via trilium-windows-xxx.zip or via chrome.

?

**Comment by meichthys** [comment]
There's a few things here:

1. There is a windows exe in the release assets (https://github.com/TriliumNext/Notes/releases/tag/v0.90.12)
2. There is no "professional" way to run trilium. All methods are valid. Just make sure you have your document (SQLite db) backed up.
3. If you want to sync multiple instances you need to expose the "server" machine (it can be a windows machine if you want it to be) to each client - there are many ways to do this but they can be more advanced and require additional technical knowledge (nginx proxy, port forwarding, VPN, etc) if you want to access your Trilium data remotely. 
4. My preferred method is to run a triliumnext docker "server" instance (which also provides a web interface), then install client instances on all my other devices (windows, macos, linux, android etc) and have those instances sync to the server (I use nginx reverse proxy, cloudflare DNS, let's encrypt). If you prefer not to have to run a docker instance yourself and deal with setting up remote access, you could rent a server from something like PikaPods: https://www.pikapods.com/apps#documentation

**Comment by Br0kenSilos** [comment]
Ditto to @meichthys number 4 option. This is how I run mine as well.  Most of the time I access it through the webUI on the server but sometimes I use the desktop clients as well. Keeps everything in sync and accessible. BUT, its not a requirement to run it like this. A single desktop instance is just as good. But like was pointed out, make sure you keep a backup.
